flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveTableCompactSinkITCase.java

```diff
@@ -42,17 +44,18 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 /** IT case for Hive table compaction in batch mode. */
-public class HiveTableCompactSinkITCase {
+@ExtendWith(TestLoggerExtension.class)
+class HiveTableCompactSinkITCase {
 
     @RegisterExtension
-    private static final MiniClusterExtension MINI_CLUSTER = new MiniClusterExtension();
+    public static final MiniClusterExtension MINI_CLUSTER = new MiniClusterExtension();
```

Comment:
Must it be public?

Reply:
TBH, The reason I changed it is only to follow the usage approach in the `MiniClusterExtension` class [javadoc](https://github.com/apache/flink/blob/bacdc326b58749924acbd8921d63eda06663a225/flink-test-utils-parent/flink-test-utils/src/main/java/org/apache/flink/test/junit5/MiniClusterExtension.java#L70).

At the same time, in the `architecture check` of IT case, we can see similar [requirement](https://github.com/apache/flink/blob/bacdc326b58749924acbd8921d63eda06663a225/flink-architecture-tests/flink-architecture-tests-test/src/main/java/org/apache/flink/architecture/rules/ITCaseRules.java#L67). Unfortunately, I looked at the actual check logic and didn't seem to require it to be public, which really confused me.